views/templates/Lists/index.tpl

```diff
@@ -13,28 +13,32 @@
 </div>
 
 <div class="data-container-wrapper flex-container-remainder">
+    <?php
+        $editListTitle = __('Edit this list');
+        $deleteListTitle = __('Delete this list');
+    ?>
     <?php foreach (get_data('lists') as $i => $list): ?>
-    <section id='list-data_<?= $list["uri"] ?>' class="data-container list-container">
+    <section id="list-data_<?= $list['uri'] ?>" class="data-container list-container">
         <header class="container-title">
             <h6><?= _dh($list['label']) ?></h6>
         </header>
-        <div class="container-content" id='list-elements_<?= $list['uri'] ?>'>
+        <div class="container-content" id="list-elements_<?= $list['uri'] ?>">
             <ol>
                 <?php foreach ($list['elements'] as $level => $element): ?>
                 <li id="list-element_<?= $level ?>">
-                    <span class="list-element" id="list-element_<?= $level ?>_<?= $element['uri'] ?>"><?=_dh($element['label']) ?></span>
+                    <span class="list-element" id="list-element_<?= $level ?>_<?= $element['uri'] ?>"><?= _dh($element['label']) ?></span>
                 </li>
                 <?php endforeach ?>
             </ol>
         </div>
         <footer class="data-container-footer action-bar  <?php !$list['editable'] && print 'hidden'?>">
         <?php if ($list['editable']): ?>
-            <button type="button" title="<?= __('Edit this list') ?>" class="icon-edit list-edit-btn btn-info small rgt" data-uri="<?= $list['uri'] ?>">
+            <button type="button" title="<?= $editListTitle ?>" class="icon-edit list-edit-btn btn-info small rgt" data-uri="<?= $list['uri'] ?>">
```

Comment:
May I ask why?

Reply:
To avoid translations on every `foreach` iteration 🙂